### ISSUE TITLE
Swift-version

### ISSUE THREAD

**Issue by lmr520** [SDWebImage/SDWebImage#3055]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           ios
 Platform Version        12.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps
[!] The following Swift pods cannot yet be integrated as static libraries:
erro:
The Swift pod `DKPhotoGallery` depends upon `SDWebImage` and `SDWebImageFLPlugin`, which do not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.

**Comment by dreampiggy** [comment]
Just follow that description, use modular-headers to enable compatible for Swift.

See readme again about CocoaPods install.
